recipes-qt/qt5/qt5-git.inc

```diff
@@ -2,7 +2,8 @@
 # Copyright (C) 2013-2018 Martin Jansa <martin.jansa@gmail.com>
 
 QT_MODULE ?= "${BPN}"
-QT_MODULE_BRANCH ?= "5.10.1"
+# branch named 5.10.1 got deleted upstream, but v5.10.1 tag still exists on 5.11 branch
+QT_MODULE_BRANCH ?= "5.11"
```

Comment:
Couldn't it use 5.10 branch here instead? The 5.10 should have all point releases merged.

Reply:
No we cannot, see my review on the ML, I'm waiting for this one hoping that v5.10.1 tag will soon be merged to 5.10 branch (like 5.11.1 and 5.9.6 were downmerged to 5.11 and 5.9 branches relatively soon after the release, but it didn't happen yet with v5.10.1 :/ )

Reply:
That is indeed ridiculous. They are doing orphan releases :-1: 

I will try to contact someone inside Qt to see if it could be handled soon.

Reply:
It has been a while since upstream broke this checkout, and the "stable" branch has been broken for anyone attempting a from-scratch build (not having earlier downloads to rely on) ever since then. 

For that reason, wouldn't it be a good idea to un-break sumo now rather than waiting longer for upstream to fix this? It doesn't need to go back to master, just a sumo-specific patch.

Reply:
I sent an e-mail to someone asking for guidance. Let's see how it evolves.

Reply:
@sapiippo Could you help with this in Qt side?

Reply:
@thiagomacieira said we should indeed use 5.11. It is indeed unfortunate but I think we will need to live with that.

Reply:
That completely breaks AUTOREV setups, because all revisions from 5.11 branch aren't compatible with the patches we have in meta-qt5 for 5.10.1.. sigh. Luckily I don't care about sumo/5.10 and Qt Company probably also doesn't when they don't downmerge their release branch.

Next time I'll be more careful in accepting point upgrades before the downmerge is finished.

Reply:
I fully agree. Qt seems to not care as well.

We need to check that the release has been merged before accept bumps.